license-compliance/gradle-license-report/license-normalizer-bundle.json

```diff
@@ -151,6 +246,130 @@
       {
         "bundleName": "mpl1.1",
         "licenseUrlPattern": "^https://www.mozilla.org/MPL/MPL-1\\.1\\.txt$"
+      },
+      {
+        "bundleName": "mpl2.0",
+        "licenseNamePattern": "^Mozilla Public License Version 2\\.0$"
+      },
+      {
+        "bundleName": "upl1.0",
+        "licenseNamePattern": "^UPL-1\\.0"
+      },
+      {
+        "bundleName": "edl1.0",
+        "licenseNamePattern": "^Eclipse Distribution License \\(New BSD License\\)$"
+      },
+      {
+        "bundleName": "epl1.0",
+        "licenseNamePattern": "^Eclipse Public License v1\\.0$"
+      },
+      {
+        "bundleName": "epl1.0",
+        "licenseNamePattern": "^Eclipse Distribution License v\\. 1\\.0$"
+      },
+      {
+        "bundleName": "bsd3clause",
+        "licenseNamePattern": "^BSD-3-Clause$"
+      },
+      {
+        "bundleName": "bsd-0",
+        "licenseNamePattern": "^BSD Zero Clause License$"
+      },
+      {
+        "bundleName": "isc",
+        "licenseNamePattern": "^ISC$"
+      },
+      {
+        "bundleName": "bsd2clause",
+        "licenseNamePattern": "^Simplified BSD$"
+      },
```

Comment:
Please remove it, because we are not sure

Reply:
removed